src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -0,0 +1,627 @@
+
+                    ssrcs_.remove((Long) Key);
+                }
+            }
+            if (ssrcs_.isEmpty()) {
+                // We can't update the estimate if we don't have any active streams.
+                interArrival = new InterArrival((kTimestampGroupLengthMs
+                        << kInterArrivalShift) / 1000,
+                        kTimestampToMs, true);
+                estimator = new OveruseEstimator(new OverUseDetectorOptions());
+                // We deliberately don't reset the first_packet_time_ms_
+                // here for now since we only probe for bandwidth in the
+                // beginning of a call right now.
+            }
+        }
+    }
+
+    void OnRttUpdate(long avg_rtt_ms,
+                     long max_rtt_ms)
+    {
+        synchronized (critSect) {
+            remoteRate.setRtt(avg_rtt_ms);
+        }
+    }
+
+    void removeStream(long ssrc)
+    {
+        synchronized (critSect)
+        {
+            ssrcs_.remove(ssrc);
+        }
+    }
+
+    boolean latestEstimate(List<Long> ssrcs,
+        long bitrateBps) {
+        // Currently accessed from both the process thread (see
+        // ModuleRtpRtcpImpl::Process()) and the configuration thread (see
+        // Call::GetStats()). Should in the future only be accessed from
+        // a single thread.
+        synchronized (critSect)
+        {
+            //RTC_DCHECK(ssrcs);
+            //RTC_DCHECK(bitrate_bps);
+            //rtc::CritScope lock(&crit_);
+            if (!remoteRate.isValidEstimate()) {
+                return false;
+            }
+            ssrcs = Keys(ssrcs_);
+            if (ssrcs_.isEmpty()) {
+                bitrateBps = 0;
+            } else {
+                bitrateBps = remoteRate.getLatestEstimate();
+            }
+            return true;
+        }
+    }
+
+    /**
+     * Gets the <tt>PacketTransformer</tt> for RTP packets.
+     *
+     * @return the <tt>PacketTransformer</tt> for RTP packets
+     */
+    @Override
+    public PacketTransformer getRTPTransformer() {
+        return null;
+    }
+
+    /**
+     * Gets the <tt>PacketTransformer</tt> for RTCP packets.
+     *
+     * @return the <tt>PacketTransformer</tt> for RTCP packets
+     */
+    @Override
+    public PacketTransformer getRTCPTransformer() {
+        return null;
+    }
+
+    /**
+     * Returns the estimated payload bitrate in bits per second if a valid
+     * estimate exists; otherwise, <tt>-1</tt>.
+     *
+     * @return the estimated payload bitrate in bits per seconds if a valid
+     * estimate exists; otherwise, <tt>-1</tt>
+     */
+    @Override
+    public long getLatestEstimate() {
+        return 0;
+    }
+
+    @Override
+    public Collection<Integer> getSsrcs() {
+        return null;
+    }
+
+    /**
+     * Removes all data for <tt>ssrc</tt>.
+     *
+     * @param ssrc
+     */
+    @Override
+    public void removeStream(int ssrc)
+    {
+
```

Comment:
In the `[webrtc RemoteBitrateEstimator.h - line 57](https://chromium.googlesource.com/external/webrtc/+/master/webrtc/modules/remote_bitrate_estimator/include/remote_bitrate_estimator.h)`, it is commented as   `// Removes all data for |ssrc|.` Am not sure if am to just remove the `ssrc` value from the list (which is redundant) here or if this is supposed to be equivalent to `removeMediaStream` in the `transportCCEngine`.